psychotropic/cogs/structuregame.py

```diff
@@ -261,7 +261,10 @@ def __init__(self, game, interaction):
         self.channel = interaction.channel
         self.start_time = datetime.now()
         self.tasks = set()
-        self.registry[self.channel.id] = self
+        if self.channel.id in self.registry.keys():
+            self.registry[self.channel.id] += [self,]
+        else:
+            self.registry[self.channel.id] = [self,]
```

Comment:
I feel ankward about the data model here: there must be at most one `RunningGame` per channel, hence storing lists in the registry seems weird